src/utils/__tests__/paths.spec.ts

```diff
@@ -130,7 +135,7 @@ describe('paths', () => {
   });
 
   it('lima should be in one of the main subtrees', () => {
-    const pathsToDelete = [paths.cache, paths.appHome, paths.config, paths.logs];
+    const pathsToDelete = [paths.cache, paths.appHome, paths.altAppHome, paths.config, paths.logs];
```

Comment:
This test breaks with the addition of `paths.altAppHome`. `pathsToDelete.some()` on line `142` becomes truthy and the tests are running on linux, so `toEqual(platform === 'darwin')` evaluates to false. 

If we expect this test to pass on Linux, then we need to update line `142` to something like

```
expect(pathsToDelete.some( dir => paths.lima.startsWith(dir))).toBeTruthy();
```

Reply:
Makes sense, thanks for the detailed explanation!

Fixing.